internal/testutil/orchestrator.go

```diff
@@ -0,0 +1,33 @@
+package testutil
+
+import (
+	"time"
+
+	"clouditor.io/clouditor/api/orchestrator"
+)
+
+// CreateCertificateMock creates a mock certificate creation request
+func CreateCertificateMock() *orchestrator.Certificate {
```

Comment:
Are multiple packages using this? otherwise I would move it to the corresponding _test file for now. Also a simple `var` would do. not sure why this should be a function

Reply:
Yes, multiple packages are using it: Tests in orchestrator and gorm. But due to cyclic dependencies I created orchestrator directory/package within testutil.
The idea behind the function is to create multiple mock certificates with different properties when needed. When you have a variable, you would have to make a copy I guess each time?